distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -1118,15 +928,24 @@ public static String getSimpleXml(String uri) {
                 ImmutableMap.of("uri", uri));
     }
 
-    public void configureShowInvalidMetacardsReset() throws IOException {
-        configureShowInvalidMetacards("false", "true", getAdminConfig());
+    /**
+     * Setting configurations is performed asynchronously and there is no way to check if the configured
+     * bean has received a configuration update. This method provides a best effort workaround by retrying
```

Comment:
❓ Should we mention that `Runnable.run()` will be retried until it it doesn't throw an exception?